plutus-tx-plugin/test/CallTrace/OtherModule.hs

```diff
@@ -0,0 +1,23 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module CallTrace.OtherModule where
+
+import PlutusTx.Prelude
+import Prelude ()
```

Comment:
```suggestion
```

Reply:
Again, we need this because we don't have `NoImplicitPrelude`

Reply:
Ah, necessity to add **explicitly** add `NoImplicitPrelude` to make prelude not implicit!

I generally not prefer adding extra language extension, but I don't really mind here.